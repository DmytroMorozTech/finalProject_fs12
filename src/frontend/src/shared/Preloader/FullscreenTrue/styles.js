import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  fullscreenTrue: {
    height: '100vh',
    width: '100vw',
    backgroundColor: theme.palette.background.default,
    position: 'absolute',
    top: 0,
    left: 0,
    zIndex: 200,
    display: 'flex',
    flexDirection: 'column',
    justifyContent: 'center'
  },

  logo: {
    display: 'flex',
    justifyContent: 'center',
    marginBottom: theme.spacing(2)
  },

  preloader: {
    width: '100px',
    margin: '0 auto'
  }
}))
