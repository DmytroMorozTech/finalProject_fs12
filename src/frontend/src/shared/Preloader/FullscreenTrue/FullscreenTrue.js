import LinkedinLogo from '../../LinkedinLogo/LinkedinLogo'
import { LinearProgress } from '@material-ui/core'
import styles from './styles'

function FullscreenTrue () {
  const classes = styles()

  return (
    <div className={classes.fullscreenTrue}>
      <div className={classes.logo}>
        <LinkedinLogo/>
      </div>
      <div className={classes.preloader}>
        <LinearProgress/>
      </div>
    </div>
  )
}

export default FullscreenTrue
