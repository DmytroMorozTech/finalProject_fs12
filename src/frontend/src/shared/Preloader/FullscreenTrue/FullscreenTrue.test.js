import React from 'react'
import FullscreenTrue from './FullscreenTrue'
import { shallow } from 'enzyme'

const setUp = (props) => shallow(<FullscreenTrue {...props}/>)

describe('should render FullscreenTrue component', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
