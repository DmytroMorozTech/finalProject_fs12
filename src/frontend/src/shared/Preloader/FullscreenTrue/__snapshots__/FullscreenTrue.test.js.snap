// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FullscreenTrue component to match snapshot 1`] = `
<div
  className="makeStyles-fullscreenTrue-1"
>
  <div
    className="makeStyles-logo-2"
  >
    <LinkedinLogo />
  </div>
  <div
    className="makeStyles-preloader-3"
  >
    <WithStyles(ForwardRef(LinearProgress)) />
  </div>
</div>
`;
